Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -9,18 +9,30 @@ import OnePasswordExtension
 private let log = Logger.browserLogger
 
 class ShareExtensionHelper: NSObject {
+    
+    enum ShareActivityType {
+        case password
+        case iBooks
+        case openByCopy
+        case `default`
+    }
+    
     fileprivate weak var selectedTab: Tab?
 
     fileprivate let selectedURL: URL
     fileprivate var onePasswordExtensionItem: NSExtensionItem!
     fileprivate let browserFillIdentifier = "org.appextension.fill-browser-action"
 
+    fileprivate func isFile(url: URL) -> Bool { url.scheme == "file" }
```

Comment:
maybe it's worth adding a helper method in a URL extension, we have `URLExtensions` file for it

Reply:
Good idea !